tests/contrib/gevent/test_tracer.py

```diff
@@ -40,75 +40,6 @@ def tearDown(self):
         # untrace gevent
         unpatch()
 
-    def test_main_greenlet(self):
-        # the main greenlet must not be affected by the tracer
-        main_greenlet = gevent.getcurrent()
-        ctx = getattr(main_greenlet, "__datadog_context", None)
-        assert ctx is None
-
-    def test_main_greenlet_context(self):
-        # the main greenlet must have a ``Context`` if called
-        ctx_tracer = self.tracer.get_call_context()
-        main_greenlet = gevent.getcurrent()
-        ctx_greenlet = getattr(main_greenlet, "__datadog_context", None)
-        assert ctx_tracer is ctx_greenlet
-        assert len(ctx_tracer._trace) == 0
-
-    def test_get_call_context(self):
```

Comment:
`test_trace_greenlet` is a very similar test which provides a stronger assertion